package.json

```diff
@@ -2,14 +2,48 @@
   "name": "bpmn-visualization",
   "version": "0.3.1",
   "description": "A TypeScript library to visualize process execution data on BPMN diagrams",
-  "repository": "https://github.com/process-analytics/bpmn-visualization-js",
+  "keywords": [
+    "bpmn2",
+    "bpmn",
+    "javascript",
+    "typescript",
+    "ui-components",
+    "diagrams",
+    "visualization",
+    "process-analytics",
+    "view",
+    "bpmn-support",
+    "analytics",
+    "diagram",
+    "viewer"
+  ],
+  "homepage": "https://github.com/process-analytics/bpmn-visualization-js#readme",
+  "bugs": "https://github.com/process-analytics/bpmn-visualization-js/issues" ,
   "license": "Apache-2.0",
-  "browser": "dist/bpmn-visualization.js",
-  "module": "dist/index.es.js",
+  "contributors": [
```

Comment:
:question: do we want to maintain this list?

Reply:
I don't think the list changes a lot.
If I understand the npm documentation, it's just the top-level maintainers : https://docs.npmjs.com/files/package.json#people-fields-author-contributors

Reply:
> npm also sets a top-level “maintainers” field with your npm user info.
Ok let's start with this list then :smiley: